core/src/hashes/blake2b256.rs

```diff
@@ -1,46 +1,18 @@
+use blake2_rfc::blake2b::Blake2b;
+use std::io::{Read, Result as IOResult, Write};
+use serde::{Deserialize, Serialize};
+
 use crate::{
     hashes::marked::{Digest, MarkedDigestWriter},
     ser::{ByteFormat, SerError, SerResult},
 };
-use blake2_rfc::blake2b::Blake2b;
-use std::io::{Read, Result as IOResult, Write};
 
 /// A blake2b256 digest.
-#[derive(Copy, Clone, PartialEq, Eq, Default, Hash)]
+#[derive(Copy, Clone, PartialEq, Eq, Default, Hash, PartialOrd, Ord, Serialize, Deserialize)]
```

Comment:
this will result in a pretty ugly serialization. can you check out the way it's done in `bitcoin-spv/src/std_types.rs`

Reply:
there's also `impl_hex_serde`